applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -398,6 +415,48 @@ static struct bt_audio_stream_ops stream_ops = {
 	.recv = stream_recv_cb,
 };
 
+static void work_stream_start(struct k_work *work)
+{
+	int ret;
+	struct worker_data work_data;
+
+	ret = k_msgq_get(&kwork_msgq, &work_data, K_NO_WAIT);
+	if (ret) {
+		LOG_ERR("Cannot get info for start stream");
```

Comment:
If there is nothing in the msg queue, should we return here?

Reply:
Agree, otherwise you would be working on unknown data